admins/pageflow/accounts.rb

```diff
@@ -16,23 +16,28 @@ module Pageflow
         end
       end
       column :entries_count do |account|
-        account.entries_count
+        account.entries_count if authorized?(:read, account)
       end
       column :users_count do |account|
-        account.memberships.size
+        account.memberships.size if authorized?(:read, account)
+      end
+      account_memberships = Membership.on_accounts.where(user: current_user)
+      account_roles = {}
+      account_memberships.each do |membership|
+        account_roles.merge!(membership.entity_id => membership.role)
       end
```

Comment:
Can be shortened a bit by using `current_user.memberships.on_account` and `each_with_object` or `reduce`.